debs-to-download

```diff
@@ -5,6 +5,8 @@ revpi-tools
 revpi-webstatus
 revpi7
 pictory
+libtss2-dev
+libtss2-esys0
```

Comment:
Why is this library added? If it is needed for piserial, it should be installed as a dependency of it.